packages/app-mobile/components/screens/Note/commands/attachFile.ts

```diff
@@ -44,7 +44,13 @@ export const runtime = (props: CommandRuntimeProps): CommandRuntime => {
 	};
 	const attachPhoto = async () => {
 		// the selection Limit should be specified. I think 200 is enough?
-		const response: ImagePickerResponse = await launchImageLibrary({ mediaType: 'photo', includeBase64: false, selectionLimit: 200 });
+		const response: ImagePickerResponse = await launchImageLibrary({
+			mediaType: 'mixed',
+			videoQuality: 'low',
+			formatAsMp4: true,
```

Comment:
It may make sense to select different values for these options. For example, if Electron on non-MacOS systems supports .mov videos, removing the `formatAsMp4` could make sense.